### PR TITLE
misc: add CODEOWNERS

### PR DESCRIPTION
**Pull request by zhyncs**
## Checklist

- [ ] Format your code according to the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [ ] Add unit tests as outlined in the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [ ] Update documentation as needed, including docstrings or example tutorials.

### CODE REVIEW

**Review by Ying1123 — APPROVED** [review]
Overall LGTM. We can always update it over the time. But let's wait for everyone's approval.

**Review by ByronHsu — APPROVED** [review]
LGTM. I can add `/python/sglang/srt/router` after https://github.com/sgl-project/sglang/pull/1695 is merged

**Review by merrymercy — APPROVED** [review]

**Review by ispobock — APPROVED** [review]

**Pull request merged**
